docs/Contributing/eval-pull-request.rst

```diff
@@ -0,0 +1,79 @@
+Evaluating a Pull Request
+*************************
+
+This page mostly targets the Longturn Admins who will be asked to review (evaluate) a submitted Pull Request.
+It is the policy of the Longturn community that all (100%) Pull Requests have at least one reviewer complete
+an evaluation of a change and either approve or make suggestions for improvements before a merge into the
+master branch.
+
+This page assumes the user knows how to use :file:`git`, compile Freeciv21 and use GitHub.
+
+:strong:`Update Local Sandbox`
+
+.. code-block:: rst
+
+  $ git checkout master
+  $ git fetch upstream master
+  $ git pull upstream master --ff-only
+
+
+:strong:`Create A Testing Branch`
+
+.. code-block:: rst
+
+  $ git checkout -b testing/pr_[pr-number] upstream/master
+
+
+:strong:`Get A Copy Of The Pull Request`
+
+.. code-block:: rst
+
+  $ wget https://github.com/longturn/freeciv21/pull/[pr-number].diff -O pr[pr-number].diff
+
+
+:strong:`Apply The Downloaded Update To The Local Sandbox`
+
+.. code-block:: rst
+
+  $ patch -p1 < pr[pr-number].diff
+
+
+:strong:`Ensure The Build Directory Is Empty`
+
+.. code-block:: rst
+
+  $ rm -Rf build
+
+
+:strong:`Configure And Compile The Code`
+
+.. code-block:: rst
+
+  $ cmake . -B build -G Ninja -DCMAKE_INSTALL_PREFIX=$PWD/build/install
+  $ cmake --build build
+  $ cmake --build build --target install
+  $ cmake --build build --target package      # MSys2 Only
+  $ cmake --build build --target docs
```

Comment:
I guess here we can do only the relevant parts since the CI tests the rest. When evaluating docs, we have RTD so we don't even need to download the patch.

Reply:
docs removed, allow CI to do it. diff is really for actual code.